testsuite/libra-fuzzer/src/fuzz_targets/executor.rs

```diff
@@ -31,3 +54,9 @@ prop_compose! {
         (txn_list_with_proof, verified_target_li)
     }
 }
+
+prop_compose! {
+    fn execute_and_commit_blocks_input()(blocks in vec((any::<HashValue>(), vec(any::<Transaction>(), 0..10)), 0..10), li_with_sigs in any::<LedgerInfoWithSignatures>()) -> (Vec<(HashValue, Vec<Transaction>)>, LedgerInfoWithSignatures) {
```

Comment:
Can you manually fix the argument indentation for this? It seems like cargo fmt doesn't handle stuff inside prop_compose.. e.g.,

fn execute_and_commit_blocks_input()(
    blocks in vec(...),
    li_with_sigs in any...,
) -> (....) {

Reply:
yes, it ignores everything in macro.